Makefile

```diff
@@ -99,6 +99,9 @@ kind-cluster: ## Standup a kind cluster for e2e testing usage
 
 install: generate kustomize cert-mgr ## Install rukpak
 	$(KUSTOMIZE) build manifests | kubectl apply -f -
+	kubectl wait --for=condition=Available --namespace=rukpak-system deployment/plain-provisioner --timeout=60s
+	kubectl wait --for=condition=Available --namespace=rukpak-system deployment/rukpak-core-webhook --timeout=60s
+	kubectl wait --for=condition=Available --namespace=crdvalidator-system deployment/crd-validation-webhook --timeout=60s
```

Comment:
Maybe we could make a variable called `NAMESPACE` and then reference it here?

Reply:
That's fair, lets do that